src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster-review.component.ts

```diff
@@ -0,0 +1,88 @@
+import { Component, OnInit } from '@angular/core';
+
+import _ from 'lodash';
+
+import { HostService } from '~/app/shared/api/host.service';
+import { CellTemplate } from '~/app/shared/enum/cell-template.enum';
+import { Permissions } from '~/app/shared/models/permissions';
+
+@Component({
+  selector: 'cd-create-cluster-review',
+  templateUrl: './create-cluster-review.component.html',
+  styleUrls: ['./create-cluster-review.component.scss']
+})
+export class CreateClusterReviewComponent implements OnInit {
+  hosts: object[] = [];
+  hostsDetails: any;
```

Comment:
```suggestion
  hostsDetails: object;
```

Reply:
It gives me error 
``` 
src/app/ceph/cluster/create-cluster/create-cluster-review.component.ts(84,25): error TS2339: Property 'data' does not exist on type 'object'.
[lint:tsc     ] src/app/ceph/cluster/create-cluster/create-cluster-review.component.ts(85,25): error TS2339: Property 'data' does not exist on type 'object'.
```
So I think I need to keep `any` type in the same way as [monitor component](https://github.com/ceph/ceph/blob/aeafb8dd4613d22e00ab9d9b2f05e0ca29cac6c5/src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/monitor/monitor.component.ts#L15)

Reply:
You have 2 options:
1) Create the appropriate data type for that var.
2) Use brackets instead of <variable>.<property>

Reply:
Good catch @alfonsomthd ! 2 comments from my end:
1. To avoid having to manually look for this, why not enabling [`no-any` tslint rule](https://palantir.github.io/tslint/rules/no-any/)?
2. I think `object` is better than `any` but not that much. @avanthakkar don't we have a `HostDetails` interface or type? Tslint `ban-types` allows [to reject certain types, like object](https://palantir.github.io/tslint/rules/ban-types/).

Reply:
I think for now declaring object type would work and using brackets instead of dots as @alfonsomthd sugeested.

Reply:
@epuertat I'm in for enforcing a rule, but not from TSLint - which is deprecated (we have to move to ESLint).
Also, enforcing this will require a dedicated PR that includes all `any` replacements.

Reply:
> I think for now declaring object type would work and using brackets instead of dots as @alfonsomthd sugeested.

@avanthakkar from a typing perspective, IMHO declaring something as `object` is just self-deception. You are not enforcing any real data structure, so there's no big gain between `any` and `object`.

@alfonsomthd the fact that TSLint is deprecated doesn't mean that the rules it enforces are deprecated too. I'd rather put this rule in place right now rather than deferring it because we have to update the whole tooling and that's not trivial... Personally I don't think we should spend time reviewing things that any tool (deprecated or not) can detect for us.